### PR TITLE
F2P Runecrafter

### PR DESCRIPTION
**Pull request by Noyan802**
Making my runecrafting script public

### CODE REVIEW

File: src/Noyan802/RuneCrafter.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+import org.powerbot.script.*;
+import org.powerbot.script.rt4.*;
+import org.powerbot.script.rt4.Bank;
+import org.powerbot.script.rt4.ClientContext;
+
+import javax.swing.*;
+import java.awt.*;
+
+@Script.Manifest(name = "Runecrafter", description ="Creates Runes", properties = "Author = Noyan802 Topic ID=1355202")
```
Comment (Strikegently) [inline src/Noyan802/RuneCrafter.java:11]: Manifest properties needs to be `topic=[thread id]` to register correctly.

**Comment by Noyan802** [comment]
I have made the correction

**Comment by Strikegently** [comment]
You just deleted every other script.

File: Noyan802/RuneCrafter.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+import org.powerbot.script.*;
+import org.powerbot.script.rt4.*;
+import org.powerbot.script.rt4.Bank;
+import org.powerbot.script.rt4.ClientContext;
+
+import javax.swing.*;
+import java.awt.*;
+
+@Script.Manifest(name = "Runecrafter", description ="Creates Runes", properties = "Author = Noyan802 topic =1355202")
```
Comment (Strikegently) [inline Noyan802/RuneCrafter.java:11]: Properties are separated by semicolons. `author=blah; topic=12345;`

**Pull request closed without merging**